source/uwp/AdaptiveCardTestApp/Views/TestResultView.xaml.cs

```diff
@@ -25,30 +25,47 @@ private void TestResultView_DataContextChanged(FrameworkElement sender, DataCont
             {
                 _currModel = model;
 
-                if (model.Status == TestStatus.Passed)
+                if (model.Status.NewCard)
                 {
-                    ButtonSaveAsExpected.Visibility = Visibility.Collapsed;
+                    // For a new card, only let them update "all"
+                    ButtonUpdateImage.Visibility = Visibility.Collapsed;
+                    ButtonUpdateJson.Visibility = Visibility.Collapsed;
+                    ButtonUpdateOriginals.Visibility = Visibility.Collapsed;
+                    ButtonUpdateAll.Visibility = Visibility.Visible;
+
+                }
+                if (model.Status.MatchedViaError)
```

Comment:
>if (model.Status.MatchedViaError) [](start = 16, length = 33)

Should this be an `else if`? I'm not sure, but seems odd having the two if statements next to each other?